en/docs/using-wso2-identity-server/oauth2-token-revocation.md

```diff
@@ -55,7 +55,7 @@ login and revoke application authorization.
 5.  Once you have logged in, click **View details** under **Authorized
     Apps**. The following page appears, listing out the available
     apps.  
-    ![]( ../../assets/img/103329616/103329617.png)
+    ![oauth-authorized-apps]( ../../assets/img/103329616/oauth-authorized-apps.png)
```

Comment:
please correct the path

Reply:
fixed with 8fce9b7e60b0c67e606677f9fc5d06b1612c3ae8